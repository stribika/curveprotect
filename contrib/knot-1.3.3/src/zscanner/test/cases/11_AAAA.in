$ORIGIN	.
$TTL	1

; OK
@	AAAA	0:0:0:0:0:0:0:0					; Minimal ipv6 address
@	AAAA	FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF		; Maximal ipv6 address
@	AAAA	\# 16 00000000 00000000 00000000 00000000	; Hexadecimal rdata
@	TYPE28	\# 16 00000000 00000000 00000000 00000000	; TYPE + Hexadecimal rdata
@	TYPE28	0:0:0:0:0:0:0:0		; TYPE
@	AAAA	0::1.2.3.4		; ipv6 address based on ipv4 address
@	AAAA	::			; Double colon
@	aaAA	::			; Type in lower-case

; KO
@	AAAA
@	AAAA				; Empty rdata
@	AAAA	\# 0			; Hex empty rdata
@	AAAA	\#			; Missing hex length
@	AAAA	0::FFFFF		; 16bit overflow
@	AAAA	0:0:0:0:0:0:0		; Short address
@	AAAA	0:0:0:0:0:0:0:X		; Bad character
@	AAAA	:: ::			; Unexpected item
@	AAAA	\# 17 00000000 00000000 00000000 00000000 00	; Too long rdata
@	AAAA	\# 17 00000000 00000000 00000000 00000000	; Bad rdata length
