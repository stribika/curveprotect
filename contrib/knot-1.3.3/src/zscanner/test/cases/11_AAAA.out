OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=00000000000000000000000001020304
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001C
RDATA=00000000000000000000000000000000
------
WARNG=ZSCANNER_EBAD_ADDRESS_CHAR
------
WARNG=ZSCANNER_EBAD_ADDRESS_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_IPV6
------
WARNG=ZSCANNER_EBAD_IPV6
------
WARNG=ZSCANNER_EBAD_ADDRESS_CHAR
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
